### PR TITLE
Fix @import tokenization

### LINKED ISSUE #42: Syntax highlighting broken for import rules
I've noticed that syntax highlighting for `@import` rules seems to be broken atm.

screen shot 2015-10-27 at 11 55 44 am

The reference url/path doesn't get highlighted at all. (not to mention the semicolon terminator)

screen shot 2015-10-27 at 11 55 23 am

Looks like it all treated as text, it's not being caught by whatever tokenizer is being used.

### PR DESCRIPTION
**Pull request by zr9**
Fixes #42

### CODE REVIEW

File: grammars/css.cson [hunk 1]
```diff
@@ -39,14 +39,26 @@
         'name': 'keyword.control.at-rule.import.css'
       '1':
         'name': 'punctuation.definition.keyword.css'
-    'end': '\\s*((?=;|\\}))'
+    'end': '\\s*(;)'
```
Comment (50Wliu) [inline grammars/css.cson]: Why can't we just match `;`?

Comment (zr9) [inline grammars/css.cson]: We can, it probably my miss after tests, should i update?

**Comment by 50Wliu** [comment]
`captures` should be renamed to `beginCaptures`.

In addition, I don't think that this actually fixes the root cause.  The real issue is the `media-query-list` repository which matches basically everything: `\\s*(?=[^{;])`.  That's overriding the strings.  The real fix would be to make `media-query-list` only match when it's supposed to.

**Comment by zr9** [comment]
Possible, but that require remove of **media-query** block and moving it under **media-query-list** otherwise we get twice repetition of condition, it's basically the reason i'm make it that way, because i'm think it done on purpose for further improvements.

**Pull request closed without merging**